mafContrib/kneMetrics.py

```diff
@@ -56,15 +60,18 @@ def interp(self, t, filtername, lc_indx=0):
 
 
 class KNePopMetric(metrics.BaseMetric):
-    def __init__(self, metricName='KNePopMetric', mjdCol='observationStartMJD', m5Col='fiveSigmaDepth',
-                 filterCol='filter', nightCol='night', ptsNeeded=2, file_list=None, mjd0=59853.5,
+    def __init__(self, metricName='KNePopMetric', mjdCol='observationStartMJD',
+                 m5Col='fiveSigmaDepth', filterCol='filter', nightCol='night',
+                 ptsNeeded=2, file_list=None, mjd0=59853.5, getLc=False,
                  **kwargs):
         maps = ['DustMap']
         self.mjdCol = mjdCol
         self.m5Col = m5Col
         self.filterCol = filterCol
         self.nightCol = nightCol
         self.ptsNeeded = ptsNeeded
+        # Boolean variable, if True the light curve will be exported
```

Comment:
Ok .. just for my sanity in the future, 'getLc' would suggest to me fetching a light curve. I think you're meaning it will be returned with the metric value? Would you mind changing this to outputLc or something similar instead?

Reply:
'getLc' is now 'outputLc'